on-prem-docs/scaling.md

```diff
@@ -0,0 +1,28 @@
+# Front-end Scaling
+With any tiered or HA deployment of the builder services you'll likely want to horizontally scale your front-end nodes. The most common deployment pattern for this usecase is a pool of front-end nodes fronted by a load-balancer.
+
+## Deploying New Front-ends
+The on-prem-builder install.sh script now supports scaling front-end nodes as a deployment pattern. It is require that new front-ends be deployed on a separate compute from your initial on-prem deployment. Similarly to Chef Automate's bootstrap pattern the on-prem builder install script can generate a bootstrap bundle which is used to simplify the deployment of new front-ends.
```

Comment:
Is there guidance on what provisioning requirements we have for FE nodes? I see we have guidance for the system in general, but might be worth calling out minimum suggested system.